src/App.vue

```diff
@@ -1185,33 +1179,6 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return userInfo
   }
 
-  /** Gets Keycloak roles, stores them, and returns them. */
-  private async loadKeycloakRoles (): Promise<string[]> {
-    const jwt = getJWT()
-    const keycloakRoles = (jwt.roles || []) as Array<string>
-    if (keycloakRoles.length < 1) throw new Error('Invalid Keycloak roles')
-    this.setKeycloakRoles(keycloakRoles)
```

Comment:
The Auth API `authorizations` call returns the list of Keycloak roles including some "row level permissions", so it's better to use that instead.